keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -403,7 +389,7 @@ _This section must be completed when targeting alpha to a release._
 
 * **How can this feature be enabled / disabled in a live cluster?**
   - [X] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name: SELinuxRelabelPolicy
+    - Feature gate name: `SELinuxMountReadWriteOncePod`
```

Comment:
Is this the final feature gate or while the feature works only for `ReadWriteOncePod` pods?

Reply:
Yes, it's final feature gate, we need another one for all volumes. It's called `SELinuxMount` below.